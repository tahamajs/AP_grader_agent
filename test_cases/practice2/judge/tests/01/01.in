show_table_info 3
reserve_table 9 3
show_table_info 3
reserve_table 4 3
show_table_info 3
reserve_table 7 3
show_table_info 3
exit 9
show_table_info 3
exit 7
show_table_info 3
show_table_info 6
exit 4
show_table_info 3
