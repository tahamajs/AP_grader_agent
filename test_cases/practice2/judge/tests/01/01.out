Table ID: 3
People at the table:
Table remaining capacity: 2
Waiting queue length: 0
Sadegh sits at table 3
Table ID: 3
People at the table: Sadegh
Table remaining capacity: 1
Waiting queue length: 0
Mehrad sits at table 3
Table ID: 3
People at the table: Mehrad, Sadegh
Table remaining capacity: 0
Waiting queue length: 0
Mahdi enters the waiting queue of table 3
Table ID: 3
People at the table: Mehrad, Sadegh
Table remaining capacity: 0
Waiting queue length: 1
Sadegh exits!
Table ID: 3
People at the table: Mahdi, Mehrad
Table remaining capacity: 0
Waiting queue length: 0
Mahdi exits!
Table ID: 3
People at the table: Mehrad
Table remaining capacity: 1
Waiting queue length: 0
Table ID: 6
People at the table:
Table remaining capacity: 3
Waiting queue length: 0
Mehrad exits!
Table ID: 3
People at the table:
Table remaining capacity: 2
Waiting queue length: 0
